packages/js-sdk/tests/sandbox/timeout.test.ts

```diff
@@ -10,14 +10,22 @@ sandboxTest.skipIf(isDebug)('shorten timeout', async ({ sandbox }) => {
   expect(await sandbox.isRunning({ requestTimeoutMs: 1000 })).toBeFalsy()
 })
 
-sandboxTest.skipIf(isDebug)('shorten then lenghten timeout', async ({ sandbox }) => {
-  await sandbox.setTimeout(5000)
+sandboxTest.skipIf(isDebug)(
+  'shorten then lenghten timeout',
+  async ({ sandbox }) => {
+    await sandbox.setTimeout(5000)
 
-  await wait(1000)
+    await wait(1000)
 
-  await sandbox.setTimeout(10000)
+    await sandbox.setTimeout(10000)
 
-  await wait(6000)
+    await wait(6000)
+
+    await sandbox.isRunning()
+  }
+)
 
-  await sandbox.isRunning()
+sandboxTest.skipIf(isDebug)('get sandbox timeout', async ({ sandbox }) => {
```

Comment:
some assertions around the time left after a waiting for a period of time with reasonable margins could be insightful here?

Reply:
my test pass